Sources/Tracing/SpanProtocol.swift

```diff
@@ -154,17 +148,28 @@ public struct SpanEvent: Equatable {
     public var attributes: SpanAttributes
 
     /// The `DispatchWallTime` at which this event occurred.
-    public let time: DispatchWallTime
+    public let time: UInt64
```

Comment:
Can we add some docs what this timestamp is e.g. millis since Epoch? Interestingly enough should we make this opaque or not?

Reply:
Opaque is rather tricky what it would mean... Practically speaking we need timestamps, a tracer could decide it needs some specific way of those but by the end of the day it all ends up as ints I guess.

Making it all a type bound to `Tracer` is a huge mess type wise if we need to retain the 5.6 support :-\